CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Changelog
+# 1.85.2
+Bugfix - Disable Digest Resolution for Manifest V2 Schema 1 [#4249](https://github.com/aws/amazon-ecs-agent/pull/4249)
```

Comment:
#4249 is not really a bugfix. It's a mitigation for a bug in ECR.

Reply:
Not sure what else we can label it though.

Reply:
agreed. 

"Mitigation" ? It would be a first in the changelog.